packages/kbn-ftr-common-functional-ui-services/services/toasts.ts

```diff
@@ -85,26 +122,36 @@ export class ToastsService extends FtrService {
     });
   }
 
-  public async getToastElement(index: number) {
-    const list = await this.getGlobalToastList();
-    return await list.findByCssSelector(`.euiToast:nth-child(${index})`);
+  public async checkForReportingToasts(): Promise<boolean> {
+    this.log.debug('Reporting:checkForReportingToasts');
+    const isToastPresent = await this.testSubjects.exists('completeReportSuccess', {
+      allowHidden: true,
+      timeout: 90000,
+    });
+    // Close toast so it doesn't obscure the UI.
+    if (isToastPresent) await this.testSubjects.click('completeReportSuccess > toastCloseButton');
+
+    return isToastPresent;
+  }
+  public async getToastElementByIndex(index: number): Promise<WebElementWrapper> {
+    return await (
+      await this.getGlobalToastList()
+    ).findByCssSelector(`.euiToast:nth-child(${index})`);
   }
 
-  public async getToastContent(index: number) {
-    const elem = await this.getToastElement(index);
-    return await elem.getVisibleText();
+  public async getToastContent(index: number): Promise<string> {
```

Comment:
You've renamed `getToastElement` to `getToastElementByIndex`. Consequentially, you should also rename `getToastContent` to `getToastContentByIndex` IMO.